docker-compose.yml

```diff
@@ -7,9 +7,9 @@ services:
     volumes:
       - ./src:/usr/src/nijobs-be/src
     ports:
-      - ${HOST_PORT}:3000
+      - ${PORT}:${PORT}
     environment:
-      - PORT=3000
+      - PORT=${PORT}
       - NODE_ENV=prod
       - DB_HOSTNAME=mongo
```

Comment:
Same suggestion as with the Travis configs above, switch this to use `MONGO_URI` as `DB_HOSTNAME` is no longer being used, I think :sweat_smile: